src/param/param_slash.c

```diff
@@ -30,6 +30,49 @@
 static char queue_buf[PARAM_SERVER_MTU];
 param_queue_t param_queue = { .buffer = queue_buf, .buffer_size = PARAM_SERVER_MTU, .type = PARAM_QUEUE_TYPE_EMPTY, .version = 2 };
 
+static int param_slash_parse_array(char * arg, int node, param_t **param, int *offsets, int *slice_detection) {
+
+	/* Search for the '@' symbol:
+	 * Call strtok twice in order to skip the stuff head of '@' */
+	char * saveptr;
+	char * token;
+	char slice_colon;
+	
+	strtok_r(arg, "[", &saveptr);
+	token = strtok_r(NULL, "[", &saveptr);
+	if (token != NULL) {
+        /* Search for the '[' symbol: */
+		// Searches for the format digit:digit.
+		sscanf(token, "%d%c%d", &offsets[0], &slice_colon, &offsets[1]);
+		// If the input was ":4" then no offsets will be set by the first sscanf, 
+		// so we check for this and try again with another format to match.
+		if (offsets[0] == INT_MIN && offsets[1] == INT_MIN){
```

Comment:
Again the semantics on this. This function expects the caller to setup output arguments to known initial values prior to calling. This might be better adjusted in this method.